utils/doclint/dotnetXmlDocumentation.js

```diff
@@ -119,7 +119,7 @@ function _wrapAndEscape(node, maxColumns = 0) {
 
   let text = node.text;
   text = text.replace(/\[([^\]]*)\]\((.*?)\)/g, (match, linkName, linkUrl) => {
-    return `<a href="${linkUrl}">${linkName}</a>`;
+    return `<a href="${new URL(linkUrl.replace('.md', ''), 'https://playwright.dev/dotnet/docs/api/').toString()}">${linkName}</a>`;
```

Comment:
thats how we do it in Node.js: https://cs.github.com/microsoft/playwright/blob/4123a55be5c6dbba54aecc23374f1c1ffe42bbc0/utils/generate_types/index.js#L346